station-recipes-v0/src/main/java/net/modificationstation/stationapi/api/recipe/FuelRegistry.java

```diff
@@ -33,15 +49,31 @@ public static void addFuelTag(TagKey<Item> tag, int fuelTime) {
     @API
     public static void addFuelItem(Item item, int fuelTime) {
         viewInvalidated = true;
-        ITEM_FUEL_TIME.put(item, fuelTime);
+        ITEM_FUEL_TIME.getOrDefault(item, addNewMapToFuels(item)).put(0, fuelTime);
```

Comment:
Registering the entire item as fuel should ignore metadata to preserve the original behavior. This makes only meta 0 valid for fuel.

Pretty sure this could be worked around by registering metadata -1 instead and including special handling for it, because, if I remember correctly, it's already being used as "all meta" in some places.